whisper/whisper-torchserve/model/model.py

```diff
@@ -0,0 +1,56 @@
+import base64
+import multiprocessing
+import os
+import subprocess
+from typing import Dict
+
+import httpx
+import requests
+from huggingface_hub import snapshot_download
+
+TORCHSERVE_ENDPOINT = "http://0.0.0.0:8888/predictions/whisper_base"
+
+
+class Model:
+    def __init__(self, **kwargs):
+        self._data_dir = kwargs["data_dir"]
+        self._model = None
+
+    def start_tochserver(self):
+        subprocess.run(
+            [
+                "torchserve",
+                "--start",
+                "--model-store",
+                f"{self._data_dir}/model_store",
```

Comment:
This doesn't look like it's checked in. Do you need to provide a simple script to generate it or do we add it to `external_data` in the config so its downloaded?